service/src/main/resources/changelog/changesets/20230105_run_set_prepopulation_target_workflow_4.yaml

```diff
@@ -0,0 +1,227 @@
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "File"
+                        },
+                        "record_attribute": "target_workflow_4_output_file_2"
+                      },
+                      {
+                        "output_name": "target_workflow_4.output_string_6",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "String"
+                        },
+                        "record_attribute": "target_workflow_4_output_string_6"
+                      },
+                      {
+                        "output_name": "target_workflow_4.output_string_2",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "String"
+                        },
+                        "record_attribute": "target_workflow_4_output_string_2"
+                      },
+                      {
+                        "output_name": "target_workflow_4.output_string_10",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "String"
+                        },
+                        "record_attribute": "target_workflow_4_output_string_10"
+                      },
+                      {
+                        "output_name": "target_workflow_4.output_string_7",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "String"
+                        },
+                        "record_attribute": "target_workflow_4_output_string_7"
+                      },
+                      {
+                        "output_name": "target_workflow_4.output_string_1",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "String"
+                        },
+                        "record_attribute": "target_workflow_4_output_string_1"
+                      },
+                      {
+                        "output_name": "target_workflow_4.output_string_8",
+                        "output_type": {
+                          "type": "primitive",
+                          "primitive_type": "String"
+                        },
+                        "record_attribute": "target_workflow_4_output_string_8"
+                      }
+                    ]
+              - column:
+                  name: record_type
+                  value: FOO
+
+        - update:
+            tableName: run_set
+            columns:
+              - column:
+                  name: method_version_id
+                  value: 40000000-0000-0000-0000-000000000004
+            where: run_set_id='10000000-0000-0000-0000-000000000004'
```

Comment:
I might be wrong but this looks like maybe it shouldn't be necessary? The `insert` looks like it's already adding it with this method version ID?

Reply:
I think you're right. This is redundant.